style.css

```diff
@@ -0,0 +1,196 @@
+body {
+  font-family: "Courier New", Courier, monospace;
+  margin: 20px;
+  padding: 10px;
+}
+
+h1 {
+  font-family: "Pacifico", cursive;
+  font-size: 5rem;
+  transform: rotate(-5deg);
+  text-align: center;
+}
+
+h2 {
+  padding: 20px 0;
+}
+
+h3 {
+  font-family: "Pacifico", cursive;
+  font-size: 2rem;
+  text-align: center;
+  color: black;
+}
+
+p.method {
+  /* line-height: 1.6; */
+}
+nav {
+  display: flex;
+  justify-content: space-around;
+  top: 0;
+  /* background-image: linear-gradient(rgb(255, 195, 204), ivory); */
+  padding-top: 18px;
+  height: 60px;
+  font-weight: bold;
+}
+
+a {
+  color: black;
+  font-size: 1.2rem;
+  text-decoration: none;
+  position: relative;
+}
+
+a:after {
+  background: none repeat scroll 0 0 transparent;
+  top: 25px;
+  content: "";
+  display: block;
+  height: 20px;
+  left: 50%;
+  position: absolute;
+  background: gold;
+  /* background-image: linear-gradient(rgb(255, 255, 255), lightpink); */
+  transition: width 0.3s ease 0s, left 0.3s ease 0s;
+  width: 0;
+  z-index: -1;
+  /* border-radius: 5px; */
+}
+a:hover:after {
+  width: 100%;
+  left: 0%;
+}
+
+.cocktail-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(min(25em, 100%), 1fr));
+  grid-gap: 30px;
+  text-align: center;
+}
+.header-rating {
```

Comment:
Missing some whitespace between selectors all throughout the file. Try and keep formatting consistent!